.github/workflows/tools-test.yml

```diff
@@ -23,9 +23,12 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         python -m pip install tox
+        python -m pip install kconfiglib
     - name: Test backport_pr
       run: cd dist/tools/backport_pr && tox
     - name: Test compile_and_test_for_board
       run: cd dist/tools/compile_and_test_for_board && tox
     - name: Test riotctrl_shell
       run: cd dist/pythonlibs/riotctrl_shell && tox
+    - name: Test kconfig script
+      run: cd dist/tools/kconfiglib/tests && ./test.sh
```

Comment:
It would be preferable to use Python tools to test the python code. I understand that a shell script can test the whole thing, but some unit testing might also be nice to have. I also think that it's possible using pytest to test cli calls (using subprocess). This way you could have a tox config like for other tools.

Reply:
I tend to agree but I think its a "contributor" call, no?